src/search.cpp

```diff
@@ -596,19 +596,20 @@ namespace {
     int moveCount, quietCount;
 
     // Step 1. Initialize node
-    Thread* thisThread = pos.this_thread();
+    prefetch(ThisThread->materialTable[pos.material_key()]);
+    prefetch(ThisThread->pawnsTable[pos.material_key()]);
```

Comment:
If you look  carefully, you'll  see  that I had to move  the perfecth from  do_move